AcadeMeData/models.py

```diff
@@ -30,17 +25,15 @@ class User(models.Model):
     # Goto: https://docs.djangoproject.com/en/3.2/ref/contrib/auth/ for API
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
     # name = models.CharField(max_length=30, default="")  # we got the name\username from the built in user model django
-    type = models.CharField(max_length=1, choices=TYPECHOICES.choices, default=TYPECHOICES.Student)
```

Comment:
Isn't that the same change made in a former PR? 
if this PR depends on it, mention it in the description. we wouldn't want to accidentally merge this before the former.

Reply:
Mentioned it, thanks